src/parquet/file/file-serialize-test.cc

```diff
@@ -64,37 +64,45 @@ class TestSerialize : public ::testing::Test {
 };
 
 TEST_F(TestSerialize, SmallFile) {
```

Comment:
I guess for debugging reasons, it would be better if this were three separate tests that all call the same function. This should provide the codec that is failing directly in the test outputß